components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -49,21 +57,36 @@ export const CurrencyIcon: FC<CurrencyIconProps> = ({ currencyKey, type, ...rest
 		<Placeholder style={{ width: props.width, height: props.height }}>{currencyKey}</Placeholder>
 	);
 
-	if (isError) {
-		return defaultIcon;
-	}
-
 	if (type === 'token') {
```

Comment:
I discovered this prop, `token`, was actually mostly never supplied, hence the attempts to try zapper and 1inch options were being skipped..